ENV_VARS.md

```diff
@@ -8,7 +8,7 @@ Agents can be configured using environment variables:
 | Name | Description |
 |------|-------------|
 | HT_SERVICE_NAME | Identifies the service/process running e.g. "my service" |
-| HT_REPORTING_ENDPOINT | Represents the endpoint for reporting the traces e.g. http://api.traceable.ai:9411/api/v2/spans |
+| HT_REPORTING_ENDPOINT | Represents the endpoint for reporting the traces For ZIPKIN reporter type use http://api.traceable.ai:9411/api/v2/spans For OTLP reporter type use http://api.traceable.ai:4317 |
```

Comment:
This file has been modified manually and we should avoid that. Could you please move this information to the proto and generate the env vars accordingly @shashank11p ?

Reply:
This was generated by tools/env-vars/main.go. This info is already there in proto for Reporting Endpoint.